src/html/welcome.html

```diff
@@ -3,12 +3,12 @@
   <div class="navbar-header">
     <a class="navbar-brand">HCC Go</a>
   </div>
-  
```

Comment:
You have a lot of white space changes.  Can you remove them?  Look into the --ignore-whitespace option for `git`.  Also, you can you remove the white space changes from this pull request?